scripts/Source/sslThreadModel.psc

```diff
@@ -1107,6 +1109,7 @@ int function FilterAnimations()
 		endIf
 		if Config.UseStrapons && Config.RestrictStrapons && (ActorCount - Creatures) == Females && Females > 0
```

Comment:
This filter prevent the bumping air effect on animations where the female actor is in the male position and the animation actor forgot to allow the strapon for the actor position.

Is mostly related to the way "SL Animation Load" treat that option and most of those  animations have the Straight or Gay tags already